trackers/huggingface/reports/randomized_data.csv

```diff
@@ -0,0 +1,169 @@
+model_name,author,class,downloads,assembles,params,chips_used,hash,license,task,model_type,cycles,tsp_compute_latency,gpu_compute_latency,tsp_gpu_compute_ratio,tsp_estimated_e2e_latency,gpu_e2e_latency,tsp_gpu_e2e_ratio,x86_latency

```

Comment:
Do we want to check this into GitHub? Or just the huggingface repo? I dont think we should put data into github.

Reply:
This is a single CSV file fore reference as discussed. Agree that we should not put real data on github.